Api/Application/Services/EmailNotifier.cs

```diff
@@ -0,0 +1,75 @@
+using Api.Domain;
+
+using MailKit.Net.Smtp;
+using MailKit.Security;
+
+using Microsoft.IdentityModel.Tokens;
+
+using MimeKit;
+using MimeKit.Text;
+
+public class EmailNotifier : INotifier<EmailContent>
+{
+    private string _smtpServer = string.Empty;
+    private string _smtpUser = string.Empty;
+    private string _smtpPassword = string.Empty;
+    private int _smtpPort = 587;
+
+    public EmailNotifier(IConfiguration configuration)
+    {
+        ConfigSmtpServer(configuration);
+    }
+
+    public void Notify(EmailContent emailContent)
+    {
+        var email = BuildEmail(emailContent);
+        SendEmailFromSmtpServer(email);
+    }
+
+    private MimeMessage BuildEmail(EmailContent emailContent)
+    {
+        if (emailContent.To.IsNullOrEmpty())
+        {
+            throw new EmailNofiticationException("Recipient's email was not found");
+        }
+
+        var email = new MimeMessage();
+        email.From.Add(MailboxAddress.Parse(_smtpUser));
+        email.To.Add(MailboxAddress.Parse(emailContent.To));
+        email.Subject = emailContent.Subject;
+        email.Body = new TextPart(TextFormat.Plain)
+        {
+            Text = emailContent.Body
+        };
+
+        return email;
+    }
+
+    private void SendEmailFromSmtpServer(MimeMessage email)
```

Comment:
It is not better for this method to be asynchronous too

Reply:
I guess not, because there is no code that needs to be awaited

Reply:
its Okey